source/installation_guide/upgrading/from-2.x-to-3.0.rst

```diff
@@ -0,0 +1,37 @@
+=====================================
+Upgrading Dovecot from 2.x to 3.0/2.4
```

Comment:
I think this should be 2.3.x? Also in filename. Since we don't want to explain 2.[012].x upgrading here.

Reply:
Yeah. True.